doc/configuration.rst

```diff
@@ -1103,6 +1103,29 @@ NetworkUSBSDWireDevice
 A :any:`NetworkUSBSDWireDevice` resource describes a `USBSDWireDevice`_
 resource available on a remote computer.
 
+USBSDWire3Device
+~~~~~~~~~~~~~~~~
+A :any:`USBSDWire3Device` resource describes a Tizen
+`SD Wire device <https://web.archive.org/web/20240121081917/https://wiki.tizen.org/SDWire>`_.
```

Comment:
This should link to an SDWire3 website, not the original sdwire website from tizen.

Reply:
My bad. Fixed that